ivy/functional/backends/jax/device.py

```diff
@@ -60,43 +60,28 @@ def to_device(
     out: Optional[JaxArray] = None,
 ):
     if device is not None:
-        cur_dev = as_native_dev(dev(x))
+        cur_dev = ivy.as_native_dev(dev(x))
         if cur_dev != device:
-            x = jax.device_put(x, as_native_dev(device))
+            x = jax.device_put(x, ivy.as_native_dev(device))
     return x
 
 
 # this is a non-wrapped function used to place JAX arrays on respective devices,
 # since if we use to_device, it will return ivy.array which is not desirable
 def _to_device(x, device=None):
     if device is not None:
-        cur_dev = as_native_dev(dev(x))
+        cur_dev = ivy.as_native_dev(dev(x))
         if cur_dev != device:
-            x = jax.device_put(x, as_native_dev(device))
+            x = jax.device_put(x, ivy.as_native_dev(device))
     return x
 
 
-def as_ivy_dev(device, /):
```

Comment:
Seems like you've deleted the `as_ivy_dev` and `as_native_dev` functions which were to be refactored 😅
Could you please add them back and make the changes as suggested in the previous review? Thanks @abdulasiraj!

Reply:
same applies to the other backends

Reply:
done